charts/document-engine/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 - [Changelog](#changelog)
+  - [3.2.1 (2024-09-20)](#320-2024-08-29)
```

Comment:
This links to nonexistent changelog entry below. See what I did here for example: https://github.com/PSPDFKit/helm-charts/pull/30/files#diff-89bb6ce3e4fdf1f854e56aa0179a6406322e578b0fd39969019481e79430b2ceR57-R62